app_dart/lib/src/request_handlers/check_flaky_builders.dart

```diff
@@ -119,6 +120,10 @@ class CheckFlakyBuilders extends ApiRequestHandler<Body> {
     final Map<String?, PullRequest> nameToExistingPRs = await getExistingPRs(gitHub, slug);
     for (final YamlMap? flakyTarget in flakyTargets) {
       final String? builder = flakyTarget![kCiYamlTargetNameKey] as String?;
+      // If target specified ignore_flakiness, then skip.
+      if (flakyTarget[kCiYamlTargetIgnoreFlakiness] != null && flakyTarget[kCiYamlTargetIgnoreFlakiness] == true) {
```

Comment:
The issue is you want to check the properties of the target, but you're just checking the target fields here